bundle/tests/git_test.go

```diff
@@ -14,13 +14,15 @@ import (
 
 func TestGitAutoLoad(t *testing.T) {
 	b := load(t, "./autoload_git")
```

Comment:
Instead of `load` you can now directly use ` b := &bundle.Bundle{...}` fixture.

Reply:
I see what you mean. However, I wonder, if it’s better to go through load process first anyway, since that’s always applied on actual Bundle before running this mutator and thus gives more realistic test.

I’d also wary about refactoring tests and functionality in the same PR, as purpose of the tests here is to check that this change is pure refactoring and does not affect behavior in unexpected way.

Reply:
In this case, the functionality is simple and localized enough that I think we'll be fine testing only the mutator since the functionality/context is localized to the `load_git_details` mutator. 

But that's a personal preference, either way sounds reasonable enough.